### DISCUSSION TITLE
/// F0 key     F0,

### DISCUSSION THREAD

**Discussion by smallB007** [gyscos/cursive#718] [General]
Hi,
In the definition of keys, in file events.rs, there is a enum with value F0, described as a function key. I've never came across keyboard with F0. Do they exist?

**Comment by gyscos** [comment]
Hi!
Indeed, I guess there's no such thing as a F0 key. We can probably remove it.

**Comment by massimo79m** [comment]
In NCURSES, the F0 is a way to 

key = F0 + n

so if i want to get a certain key, i have to do

fn ret_key (key : usize) -> Key
{
return F0 +  key
}

i don't know if cursive works in the same way.
